apps/coa/behaviours/send-email-notification.js

```diff
@@ -0,0 +1,144 @@
+
+const config = require('../../../config');
+const utilities = require('../../../lib/utilities');
+const _ = require('lodash');
+const NotifyClient = utilities.NotifyClient;
+const userConfirmationTemplateId = config.govukNotify.userConfirmationTemplateId;
+const caseworkerConfirmationTemplateId = config.govukNotify.caseworkerConfirmationTemplateId;
+const caseworkerEmail = config.govukNotify.caseworkerEmail;
+const notifyKey = config.govukNotify.notifyApiKey;
+const translation = require('../translations/src/en/fields.json');
+const notifyClient = new NotifyClient(notifyKey);
+
+module.exports = class SendEmailConfirmation {
+  
+  async sendUserEmailNotification(req) {
+    const personalisations = this.notifyPersonalisations;
+
+    // Retrieve the value from the session model
+    const details_updating_value = req.sessionModel.get('which-details-updating');
+    
+    // Retrieve the corresponding label from the translations file
+    const details_updating_label = translation['which-details-updating'].options[details_updating_value].label;
+
+    try {
+      if (notifyKey === 'USE_MOCK') {
+        req.log('warn', '*** Notify API Key set to USE_MOCK. Ensure disabled in production! ***');
+      }
+
+      await notifyClient.sendEmail(userConfirmationTemplateId, req.sessionModel.get('email'), {
+        personalisation: Object.assign({}, personalisations, {
+        reference_number: req.sessionModel.get('uniqueRefNumber'),
+        applying_for_whom: req.sessionModel.get('isApplicant') ? 'your' : req.sessionModel.get('nameWithPossession'),
```

Comment:
`applying_for_whom` - is not used in user template. replace the `applicant_full_name `with `applying_for_whom` placeholder

You requested to update ((applicant_full_name)) details
Thank you for your request to change ((applicant_full_name)) details with UK Visas and Immigration.